cypress/locators/index.js

```diff
@@ -1,29 +1,23 @@
 import {
-  ENABLE_BACKGROUND_UI_CHECKBOX, TITLE_INPUT, SUBTILE_INPUT, LABEL_INPUT,
-  CHILDREN_TEXTAREA, HEIGHT_INPUT, LABEL_HELP_INPUT, INPUT_WIDTH_SLIDER,
-  LABEL_WIDTH_SLIDER, FIELD_HELP_INPUT, LABEL_INLINE_CHECKBOX, LABEL_ALIGN_SELECT,
+  ENABLE_BACKGROUND_UI_CHECKBOX, HEIGHT_INPUT, INPUT_WIDTH_SLIDER,
+  LABEL_WIDTH_SLIDER, LABEL_INLINE_CHECKBOX, LABEL_ALIGN_SELECT,
   HELP_ICON_PREVIEW, FIELD_HELP_PREVIEW, LABEL_HELP_PREVIEW, FORM, STORY_ROOT,
   SHOW_CLOSE_ICON_CHECKBOX, STICKY_FORM_FOOTER, DISABLE_ESC_KEY_CHECKBOX,
-  CLOSE_ICON_BUTTON, BACKGROUND_UI_LOCATOR,
+  CLOSE_ICON_BUTTON, BACKGROUND_UI_LOCATOR, LINK,
 } from './locators';
 
 // knobs locators
+export const getKnobsInput = propertyName => cy.get(`[name="${propertyName}"]`);
```

Comment:
locator should be moved to locators.js file

Reply:
export const KNOBS_NAME = propertyName => `[name="${propertyName}"]`; 

i potem użyć w index.js
export const getKnobsInput = propertyName => cy.get(KNOBS_NAME(propertyName));

Reply:
resolved